### PR TITLE
Update Ambari docs for new features

### PR DESCRIPTION
**Pull request by wolf31o2**
This removes documentation about features which were implemented during the release 3.5 timeline.

### CODE REVIEW

**Comment by awholegunch** [comment]
Running as [Quick Build 1](http://builds.cask.co/browse/CDAP-DQB104-1). Fails, but unrelated to this PR.

Page of interest: 
- [Installation using Apache Ambari](http://builds.cask.co/artifact/CDAP-DQB104/shared/build-1/Docs-HTML/3.5.0/en/admin-manual/installation/ambari.html)
- [Security](http://builds.cask.co/artifact/CDAP-DQB104/shared/build-1/Docs-HTML/3.5.0/en/admin-manual/installation/ambari.html#enabling-security)
- [HA Installation](http://builds.cask.co/artifact/CDAP-DQB104/shared/build-1/Docs-HTML/3.5.0/en/admin-manual/installation/ambari.html#enabling-cdap-ha)

File: cdap-docs/admin-manual/source/installation/ambari.rst [hunk 1]
```diff
@@ -355,7 +349,70 @@ Kerberos support in CDAP is automatically enabled when enabling Kerberos securit
...
+
+CDAP Router
+...........
+The CDAP Router service is a stateless API endpoint for CDAP, and simply routes requests to the
+appropriate service. It can be scaled horizontally for performance. A load balancer, if
+desired, can be placed in front of the nodes running the service.
+
+- Using the Ambari UI, add additional hosts for the ``CDAP Gateway/Router
+  Service`` to additional machines.
+- Start each ``CDAP Gateway/Router Service`` role.
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/installation/ambari.rst]: It's just `CDAP Router` in Ambari. I don't mention Gateway.

**Comment by wolf31o2** [comment]
We can also remove the section about SPARK_HOME (Customize CDAP 6) and Explore (Customize CDAP 8) since we always require Hive (no optional dependencies in Ambari)...

File: cdap-docs/admin-manual/source/installation/ambari.rst (hunk 1 above)
Comment (awholegunch) [reply cdap-docs/admin-manual/source/installation/ambari.rst]: Fixed.

**Comment by awholegunch** [comment]
Removed sections in CDAP no. 6 and CDAP no. 8. Please check my last two commits. LGTM.

**Comment by wolf31o2** [comment]
It looks like there may still be screenshots for these. Are you planning on just updating those with another PR, since you are going to be doing screenshots, anyway? If so, this 👍 LGTM

**Comment by awholegunch** [comment]
Yes, the screenshots are coming along elsewhere.

**Pull request merged**